.github/workflows/scheduler.yml

```diff
@@ -1,7 +1,7 @@
 name: Run a bot at a regular interval
 on:
   schedule:
-    - cron: '*/1 * * * *'
+    - cron: '* */1 * * *'
```

Comment:
The decision was events based, and not cron so this doesn't look right.

Reply:
Yes it was. but we don't have a certain event which could be triggered everyday, every hour. but please let me know the event name which would be triggered regularly. If then, this would be just replacement

Reply:
On any our "scheduled" events should piggy back off of

Reply:
Which events are scheduled ones?

Reply:
Any logic you plan to run from cron should run as a post hook handler, for example.

Reply:
The problem is that we don't know the events triggered at a regular interval. For example, we want to post weekly updates on discord/telegram. This action needs to be happened once a week. It would be best it can happen on Monday but there could be no events until Tuesday. Then weekly updates would be delayed until Tuesday. 

The bounty bot can run in both ways in general. One is the passive way and other one is active way. 
Both have their pros/cons actually but it seems you're just suggesting a passive way which means we could do nothing for which SHOULD happen at a certain time if no events emitted. 

Please clarify me again if you can. If we MUST do it only in passive way, I might need to add `research` label to this issue and spend time to find the proper events (I am not 100% sure I could find them in a short time)

Reply:
> For example, we want to post weekly updates on discord/telegram.

This is a good point and I'd like to think about this more from the perspective of scaling this to other (e.g. infinite) partner projects. The final vision will be to have some sort of matchmaking system (imagine if we have thousands of new bounties/tasks posted every week across all partner projects) in this vision, posting the new bounties every week would not make sense. 

Perhaps we can make a separate script run for this particular situation with the intent that we will deprecate it as soon as we have like 10+ partners (imagine posting a list of 50+ bounties every Monday in the chat- that would get kind of ridiculous)

Reply:
`We have a handful of events that occur on the Ubiquity Dollar repo every day actually. I try to go through the notifications at least once a day and there seems to have consistently been daily activity for weeks now, minus New Years Eve and Christmas Eve I believe.`

This is just rough estimation. I guess we couldn't rely on the irregular events for the regular activities at all.

Reply:
`Perhaps we can make a separate script run for this particular situation with the intent that we will deprecate it as soon as we have like 10+ partners (imagine posting a list of 50+ bounties every Monday in the chat- that would get kind of ridiculous)`

If you want a separate script for regular activities, we can definitely not use cron,  Just make me clear in development stage. Literally separate script is out of scope of this project then.

Reply:
@pavlovcik . would you please give me your proof why cron isn't suitable for such regular action based on your experience?

Reply:
I Google searched events based vs timetabling and there's a huge amount of articles that point to events based being superior (for most cases.) I could link you random articles from the first page of the Google results but it would be pretty easy for you to find yourself because there are so many.

In addition I asked @transitive-bullshit his thonks on this matter.

<img width="553" alt="image" src="https://user-images.githubusercontent.com/4975670/212642139-c68f9b73-e816-4de7-960b-659f2ad905b6.png">

In conclusion, cron is nice for quick hack jobs. Events-based is how robust, production-ready systems are generally built.

Reply:
no idea if cron is right for this specific use case; just that in general, I'd expect bots to respond instantly to user-defined bot events, and GH bots like vercel and https://socket.dev focus on responding via webhooks. again, no idea if that's relevant to the functionality in this specific PR. just a general precedent.

Reply:
Agree! we did a bunch of discussions already. thanks for confirmation @pavlovcik  @transitive-bullshit

Reply:
this would be updated as discussed on feat/46-supabase-adapter.